public/css/main.css

```diff
@@ -10,11 +10,19 @@
   font-style: normal;
 }
 
+html {
+  height: 100%;
+}
+
 body {
   background-image: radial-gradient(circle, #3241a6 0, #202a6b 120%);
   font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI",
     "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue",
     "Arial", sans-serif;
+  height: 100%;
+  min-height: 100vh;
+  overflow-y: auto;
+  -webkit-overflow-scrolling: touch;
```

Comment:
Nice touch, I'm not sure it's worth it for a small app like this? Personally I'm not a fan of momentum based scrolling, but can see why some people may prefer it.

Reply:
Agreed, removed.